Makefile

```diff
@@ -0,0 +1,60 @@
+.PHONY: dummy_translations extract_translations fake_translations help pull_translations push_translations \
+
+
+.DEFAULT_GOAL := help
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+quality: ## check coding style with pycodestyle and pylint
+	echo 'TODO: Refactor quality checks from .travis.yml into this Makefile'
+	exit 1
+
+requirements: ## install development environment requirements
+	pip install -r requirements.txt --exists-action w
+	pip install -r requirements-dev.txt --exists-action w
+
+test: clean ## run tests in the current virtualenv
```

Comment:
Use a similar style throughout the makefile: name, colon, comment, then the commands start on the next line. Ditto below.

Reply:
I'm using the [edX's cookiecutter](https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile) and that's a convention there, which I'd like to stay very close to. The goal is to make this XBlock as convenient to reason about as possible for other Open edX developers.